common/db_test.go

```diff
@@ -679,3 +759,19 @@ func getRowCountOrFatal(query string, dbraw *sql.DB, t *testing.T) int {
 
 	return count
 }
+
+func defaultMiniHeader(t *testing.T) *blockwatch.MiniHeader {
```

Comment:
I don't think this function needs to accept `*testing.T` as an argument.

Reply:
Fixed in ced063e